workspaces/libnpmpublish/test/publish.js

```diff
@@ -732,7 +792,12 @@ t.test('publish existing package with provenance in gha', async t => {
 
   registry.getVisibility({ spec, visibility: { public: true } })
   registry.nock.put(`/${spec.escapedName}`, body => {
-    return t.match(body, packument, 'posted packument matches expectations')
+    const bundleAttachment = body._attachments['@npmcli/libnpmpublish-test-1.0.0.sigstore']
+    const bundle = JSON.parse(bundleAttachment.data)
+    const provenance = JSON.parse(Buffer.from(bundle.dsseEnvelope.payload, 'base64').toString())
+
+    return t.match(body, packument, 'posted packument matches expectations') &&
```

Comment:
I think two `t.match` statements followed by a `return true` match more closely how we do it elsewhere.

Reply:
Actually @nlf reminds me that `t.match` has very unexpected behavior. `t.hasStrict` is more appropriate.

Reply:
Switched to `t.hasStrict` and pared things down a bit so I'm only looking at the important bits of the provenance.